fossid-webapp-client/src/main/kotlin/api/Project.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Project {
```

Comment:
Please add some documentation to this and the other public classes you add in this PR, similar to how it's done in the clearly-defined module (e.g. links to the related API docs for the classes).

Reply:
I can add the documentation but, as far as I know, there is no public documentation for the FossID RESTful API.

The only documentation is the one you get when you have an actual FossId server instance installed, but this is not public.

Reply:
Also even with the documentation they are delivering, they don't explain their data model. So aside writing in the documentation for instance "The FossId representation of a Project", I have nothing. And in this case, the documentation is useless...

Reply:
FossID team told me they will update the documentation and make it available on Internet, but without ETA.
Can I push the API "as if" without documentation and update in the future when the official documentation is available ?

Reply:
Ok, then let's add that documentation later. But the `FossId` and `RemoteScanner` classes require class docs.

Reply:
done